Runtime/Scripts/Export/GameObjectExport.cs

```diff
@@ -16,11 +16,14 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.IO.Compression;
 using System.Threading;
 using System.Threading.Tasks;
 using GLTFast.Schema;
+using UnityEditor;
```

Comment:
probably need to add `#if UNITY_EDITOR` here or compilation will fail when building

Reply:
Yep, there's a bunch of `AssetDatabase` calls I need to add that to as well, so that I can build.

Reply:
OK, I've fenced up all the editor code with the `UNITY_EDITOR` flag :construction: